### DISCUSSION TITLE
show freq instead (or in addition) of note

### DISCUSSION THREAD

**Discussion by muzzol** [ZL-Audio/ZLEqualizer#87] [Q&A] [answered]
is it possible to show frequency instead of (or side by side with) the note?

sorry if this is a basic question, just started using this plugin.

![zl](https://github.com/user-attachments/assets/f4b14c6d-4278-4df7-bc52-9c03bac2230d)

**Comment by zsliu98** [comment, marked as answer]
I prefer the pitch notation. If you do want to view the freq/gain, you could set the `Tag Colour` to non-transparent (in the UI settings). After that, the filter frequency/gain will appear as two tags on the spectrum.

**Reply by muzzol** [reply]
thanks a lot, I didn't even knew how to make UI menu appear.

this plugin is awsome.

by the way, is there any diference between lv2 and vst3 on linux in relation to performance and latency?

**Reply by zsliu98** [reply]
@muzzol The performance should be the same if the DAW hosts the plugin properly. When `Zero LAT` is off, the latency is 1 ms (e.g., for 44100Hz sample rate, latency is 44 samples). When `Zero LAT` is on, the latency is 0. BTW, I would suggest reading the [manual](https://zl-audio.github.io/plugins/zlequalizer/) before using the plugin.

**Reply by Xylemflo** [reply]
Thanks, this makes a big difference to many of us. I personally think the tags should be on by default.
